docs/quantum_chess/quantum_chess_rest_api.ipynb

```diff
@@ -0,0 +1,522 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "y_NOwwjwfqsn"
+      },
+      "source": [
+        "from recirq.quantum_chess.quantum_board import CirqBoard\n",
+        "from recirq.quantum_chess.bit_utils import ( bit_to_square, xy_to_bit )\n",
+        "from recirq.quantum_chess.move import to_rank\n",
+        "\n",
+        "global_board = CirqBoard(1)\n",
+        "\n",
+        "def print_game(board):\n",
+        "    board.print_debug_log()\n",
+        "    print('\\n')\n",
+        "    print(board)\n",
+        "    print(\"\\n\\n\")\n",
+        "\n",
+        "probs = global_board.get_probability_distribution()\n",
+        "\n",
+        "print_game(global_board)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "s-B1gYs9lo0X"
+      },
+      "source": [
+        "\n",
+        "#Quantum Chess REST API\n",
+        "The Quantum Chess REST API defines an interface for [REST](https://restfulapi.net/) endpoints that the Quantum Chess Engine can be directed to use when interacting with the quantum state of the game. The API declares an interface for three functions:\n",
+        "* init\n",
+        "* do_move\n",
+        "* undo_move\n",
+        "\n",
+        "All three endpoints must return a json object with the following values:\n",
+        "* probabilities: an array of 64 floating point numbers representing the probability of each square being occupied. Array indices are mapped to board squares starting from a1, and increasing along rows to h8.\n",
+        "\n",
+        "<center>\n",
+        "<img src='../images/chess_board_indices' width=\"300\" >\n",
+        "</center>\n",
+        "\n",
+        "* empty_bitboard: A bitboard with bits set to 1 for all squares known to be empty, i.e. 0% chance of being occupied.\n",
+        "* full_bitboard: A bitboard with bits set to 1 for all squares known to be occupied, i.e. 100% chance of being occupied.\n",
+        "\n",
+        "A [bitboard](https://www.chessprogramming.org/Bitboards) is a 64-bit integer, where each bit corresponds to a square on the chess board. The bitboard is encoded in little endian form, with the least significant bit corresponding to a1, and increasing along rows up to h8 in the most significant bit.\n",
+        "<center>\n",
+        "<img src='../images/bitboard_order.png' width=\"700\" >\n",
+        "</center>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gYNJ6tMHnyl-"
+      },
+      "source": [
+        "##init(init_basis_state) : { probabilities, empty_bitboard, full_bitboard }\n",
```

Comment:
Can you use more descriptive names for these headings? This is difficult to understand at a glance in the nav